openai-connector-test/pom.xml

```diff
@@ -39,7 +41,6 @@
 			<groupId>com.axonivy.ivy.test</groupId>
 			<artifactId>unit-tester</artifactId>
 			<version>${tester.version}</version>
-			<scope>test</scope>
```

Comment:
I don't see why this should be removed. This is a testing-only dependency, isn't it ?